src/main/resources/db/migration/all/20240411164054__add_pdu_code.sql

```diff
@@ -0,0 +1,110 @@
+ALTER TABLE probation_delivery_units ADD COLUMN pdu_delius_code TEXT;
```

Comment:
I would name it as 'delius_code' instead of 'pdu_delius_code'. Because the the `pdu` is covered in the table name `probation_delivery_unit`.

Reply:
Done 
I thought about it but to make it clear I added prefix of pdu_